### DISCUSSION TITLE
Legacy post editor

### DISCUSSION THREAD

**Discussion by aeiouaeioufootball** [AprilSylph/XKit-Rewritten#972] [Suggestion Box]
With the old post editor format intending to be fully retired, it'd be really nice if there'd be a way to toggle to use the old post editor format instead of the new one (since the new one is kind of garbage/basically unusable at least on my end, and isn't compatible with things like quicktags and etc). I imagine there may be some difficulty with post formats like the new polls since they don't have an old post editor equivalent, but besides that a general option to switch which post editor is being used would be nice. 

edit: the quicktags thing was me having both xkits installed :P but still, the rest stands.

**Comment by marcustyphoon** [comment]
> since the new one... isn't compatible with things like quicktags and etc

XKit Rewritten's Quick Tags only works in the new one, though? :D

**Reply by aeiouaeioufootball** [reply]
Huh - i might have a setting messed up there then. The rest still stands though.

**Reply by marcustyphoon** [reply]
Yeah, perhaps you have both XKit versions enabled and are still using a number of extensions from the previous one!

**Comment by aeiouaeioufootball** [comment]
the new beta editor continues to totally suck in every possible way and it seems like the toggle for the legacy editor has finally disappeared, at least on my end. this is totally unusable. 
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/125402724/9787ea73-69f8-49a3-a2f7-f4c7c7f1a1e6)
it's basically impossible to make long-form text posts with the new editor. It just totally shits the bed every time. A toggle somewhere to be able to switch back to the legacy editor, even if it doesn't have compatibility with certain types of posts like polls, would be really helpful. just so I don't completely lose my mind every time I try to post. I would happily give up polls if it meant I can actually post again.

**Reply by marcustyphoon** [reply]
Can't tell what your screenshot is supposed to be communicating?

**Reply by aeiouaeioufootball** [reply]
just that the post editor sucks and makes no sense. in this case i had tried to ctrl+A to select the text like is typical of every other website and it selected a bunch of random invisible sections instead. this was never an issue with the old post editor. I've already complained to tumblr support about this multiple times but they've just either said nothing or said the new post editor will be rolled out in full anyways.

**Reply by marcustyphoon** [reply]
If you press ctrl+A once inside a text block, it will select all of the text within that block; if you then press it a second time it will select every block. The UI for that second level of selection looks quite bad, though, yeah. But it functions correctly.
